public/assets/harvard.js

```diff
@@ -143,9 +143,30 @@ $(function() {
   }
 });
 function responsive_search(){
-  $keyword = $("select.search-keyword").children().first();
+	
+  $keyword = $("select.search-keyword").children("option:selected")
```

Comment:
$keyword (and other vars here) don't look like they have a reason to be globally scoped.

They ought to be declared - given the browser support window targeted by ASpace, I think the right thing to do would be to declare them at top of function with var.